state/endpoint_bindings_test.go

```diff
@@ -109,35 +117,38 @@ func (s *BindingsSuite) TestMergeBindings(c *gc.C) {
 		about:  "oldMap overrides defaults, newMap is nil",
 		newMap: nil,
 		oldMap: map[string]string{
-			"foo1": "client",
+			"foo1": s.clientSpaceID,
 			"self": "db",
```

Comment:
Maybe change these to some artificial space ID constant for clarity?

Reply:
this test has been updated